insights/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
you don't need separate .gitignore file for this folder, one in root of code repo should do.